src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3319,4 +3319,7 @@
   <data name="Preview" xml:space="preserve">
     <value>Preview</value>
   </data>
+  <data name="MultipleValues" xml:space="preserve">
```

Comment:
Would it be helpful to add a comment for translators to know where this is used?

Reply:
Added a comment.